src/TaskRunner/Commands/ComponentCheckCommands.php

```diff
@@ -172,6 +176,39 @@ protected function prepareSkips(): void
         }
     }
 
+    /**
+     * Validate project configuration.
+     */
+    protected function componentConfiguration()
+    {
+        // Make sure forbidden files do not exist.
+        // (because of deprecation or another reason)
+        // Get forbidden/deprecated files from configuration.
+        $files = $this->getConfig()->get('toolkit.forbidden_files');
+        // Detect forbidden files in project.
+        foreach ($files as $file) {
+            // Check if the file must be forbidden under the condition.
+            if (!empty($file['condition_callback']) && is_callable($file['condition_callback'])) {
+                // Invoke the callback method and if the condition is not met then don't forbid the file.
+                if (call_user_func_array($file['condition_callback'], []) === false) {
```

Comment:
If the parameters are not used, could this function be used instead ?
https://www.php.net/manual/en/function.call-user-func.php